aws/resource_aws_lambda_event_source_mapping_test.go

```diff
@@ -209,6 +209,43 @@ func TestAccAWSLambdaEventSourceMapping_sqs_withFunctionName(t *testing.T) {
 	})
 }
 
+func TestAccAWSLambdaEventSourceMapping_SQSBatchWindow(t *testing.T) {
```

Comment:
looks great! just a small request to update the other BatchWindow test name if you can to reflect which source it's using i.e. `TestAccAWSLambdaEventSourceMapping_BatchWindow` -> `TestAccAWSLambdaEventSourceMapping_KinesisBatchWindow`

Reply:
💯. have updated for Kinesis as well.